packages/alfa-dom/src/node/element.ts

```diff
@@ -488,21 +490,13 @@ export namespace Element {
 
         if (element.shadow.isSome()) {
           const shadow = element.shadow.get();
-          clonedElement._attachShadow(
-            Shadow.of(
-              shadow
-                .children()
-                .map((child) => Node.clone(child, options, device)),
-              shadow.style,
-              shadow.mode,
-              shadow.externalId,
-              shadow.extraData,
-            ),
-          );
+          clonedElement._attachShadow(Shadow.clone(shadow, options, device));
```

Comment:
**Nitpick:** We can inline `shadow`, to match what is done with content document.